presto-main/src/test/java/io/prestosql/operator/TestDynamicFilterSourceOperator.java

```diff
@@ -121,6 +143,28 @@ private Operator createOperator(OperatorFactory operatorFactory)
         return new DynamicFilterSourceOperator.Channel(new DynamicFilterId(Integer.toString(index)), type, index);
     }
 
+    private void assertDynamicFilters(int maxFilterPositionsCount, List<TupleDomain<DynamicFilterId>> expectedTupleDomains, List<Type> types, Page... pages)
+    {
+        assertDynamicFilters(maxFilterPositionsCount, DataSize.of(10, KILOBYTE), 1_000_000, expectedTupleDomains, types, pages);
+    }
+
+    private void assertDynamicFilters(
+            int maxFilterPositionsCount,
+            DataSize maxFilterSize,
+            int minMaxCollectionLimit,
+            List<TupleDomain<DynamicFilterId>> expectedTupleDomains,
```

Comment:
make `List<TupleDomain<DynamicFilterId>> expectedTupleDomains` last argument here and in method above